keras_hub/src/bounding_box/mask_invalid_detections.py

```diff
@@ -0,0 +1,81 @@
+from keras import ops
```

Comment:
move this into model folder

Reply:
I followed the KerasCV API and decided to expose this, as it was most likely intended to be used across all `prediction_decoder` layers. However, I can move it to the model folder instead, since it's only used for the NMS layer.